charts/diskmanager/templates/config/configMap.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.name }}-cm
+  labels:
+    {{ include "diskmanager.labels" . | nindent 4 }}
```

```suggestion
    {{- include "diskmanager.labels" . | nindent 4 }}
```
Did you confirm this renders correctly? I think this will add two newlines, the one you have here and the one added by `nindent`. The `-` ensures any preceding whitespace is gobbled up before the newline is added.